platform/mellanox/mlnx-platform-api/sonic_platform/dpuctlplat.py

```diff
@@ -169,23 +189,32 @@ def get_open_fd(self, path, flag):
         finally:
             os.close(fd)
 
+    def get_pci_dev_path(self):
```

Comment:
@gpunathilell any specific reason why this PCI remove could not be done in common code? @vvolam fyi

Reply:
we discussed this at the meeting. We have an extra dependency on the rshim here. Before removing PCI we need to stop rshim. After rescaning the PCI we need to start the rshim. Common code can't cover this.